includes/class-wp-job-manager-recaptcha.php

```diff
@@ -0,0 +1,242 @@
+<?php
+/**
+ * File containing the WP_Job_Manager_Recaptcha class.
+ *
+ * @package wp-job-manager
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Notices_Conditions_Checker class.
```

Comment:
```suggestion
 * WP_Job_Manager_Recaptcha class.
```

Reply:
Done in 7f4d9d052bc89ea9a2952811160ef6200bf0c535